src/applications/vaos/tests/new-appointment/components/CommunityCareProviderSelectionPage/index.unit.spec.jsx

```diff
@@ -247,9 +247,10 @@ describe('VAOS <CommunityCareProviderSelectionPage>', () => {
       },
     );
 
-    // Choose Provider
+    // Choose Provider that is buried 2 clicks deep
     userEvent.click(await screen.findByText(/Choose a provider/i));
     userEvent.click(await screen.findByText(/more providers$/i));
+    userEvent.click(await screen.findByText(/more providers$/i));
```

Comment:
It's weird that you had to add this, since I'm not having a problem with this test locally and haven't noticed this fail in Jenkins. But this change should make the test more reliable anyway, so I'm find with it.